listafacil-backend/.env.example

```
@@ -0,0 +1,18 @@
+HOST=127.0.0.1
+PORT=3333
```

Acho que ficaria melhor na porta 9000
```suggestion
PORT=9000
```